app/src/main/java/com/boolder/boolder/view/map/BoolderMap.kt

```diff
@@ -553,4 +553,9 @@ class BoolderMap @JvmOverloads constructor(
                 ?: listener?.onAreaLeft()
         }
     }
+
+    fun onBottomSheetHidden() {
+        unselectProblem()
+        listener?.onTopoUnselected()
+    }
```

Comment:
Let's avoid creating a new function here, the more there are, the harder it will be to understand 😅